component-matrix/pom.xml

```diff
@@ -92,7 +92,7 @@
         <version.org.jboss.logmanager.log4j-jboss-logmanager>1.1.4.Final</version.org.jboss.logmanager.log4j-jboss-logmanager>
         <version.org.jboss.marshalling.jboss-marshalling>2.0.5.Final</version.org.jboss.marshalling.jboss-marshalling>
         <version.org.jboss.modules.jboss-modules>1.8.1.Final</version.org.jboss.modules.jboss-modules>
-        <version.org.jboss.msc.jboss-msc>1.3.2.Final</version.org.jboss.msc.jboss-msc>
+        <version.org.jboss.msc.jboss-msc>1.4.1.Final</version.org.jboss.msc.jboss-msc>
```

Comment:
Could you amend you commit and reference https://issues.jboss.org/browse/WFCORE-3763 please?
JBoss MSC is a key component of WildFly and we should track its upgrade in separate WFCORE issue.

Reply:
Done @jmesnil - thanks for pointing it out.